### PR TITLE
[GraphQL] Sort by NID; up parallel requests back to 15; paginate Node QA

### PR DESCRIPTION
**Pull request by ncksllvn**
## Description
- Now that the CMS DB is upgraded, we are ready to rumble with 15 concurrent requests. 
- This PR also segments the Node QA dataset, which is not that large but updated regularly, so by segmenting it we can get some caching in play.
- This PR also sort by `nid` (entityId`), which is faster and unique, so that it produces the same sorted list each time. This resolves an issue with segmentation, https://dsva.slack.com/archives/CT4GZBM8F/p1613237269089200.

## Testing done
- Ran `yarn build:content --pull-drupal`
- Confirmed no broken links in output
- Ran the sort by `nid` in the GraphQL Explorer to confirm that successfully sort by `entityId`

<details>
<summary>output</summary>

```
Attempting to load Drupal content from API at https://prod.cms.va.gov
Pulling from Drupal via GraphQL...
Received node counts...
┌────────────────────────────┬───────┐
│          (index)           │ count │
├────────────────────────────┼───────┤
│        benefitPages        │  274  │
│           vaForm           │  438  │
│  healthCareLocalFacility   │  21   │
│   healthServicesListing    │   3   │
│           event            │  532  │
│ healthCareRegionDetailPage │  113  │
│    healthCareRegionPage    │   3   │
│       personProfile        │  124  │
│           nodeQa           │  76   │
└────────────────────────────┴───────┘
| GetTaxonomies | 0s | [n/a] pages |
| GetMenuLinks | 0s | [n/a] pages |
| GetHomepage | 0s | [n/a] pages |
| GetOutreachAssets | 0s | [n/a] pages |
| GetAlerts | 0s | [n/a] pages |
| GetBannnerAlerts | 0s | [n/a] pages |
| GetOutreachSidebar | 0s | [n/a] pages |
| GetFacilitySidebar__vaStCloudHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaSiouxFallsHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaNebraskaWesternIowaHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaMinneapolisHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaIowaCityHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaFargoHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaCentralIowaHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaBlackHillsHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaSouthernNevadaHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaSierraNevadaHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaSanFranciscoHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaPaloAltoHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaPacificIslandsHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaNorthernCaliforniaHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaWallawallaHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaSpokaneHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaCentralCaliforniaHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaRoseburgHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaSouthernOregonHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaPugetSoundHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaPortlandHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaBoiseHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaAlaskaHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaWesternColoradoHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaSheridanHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaSaltLakeCityHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaOklahomaHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaMontanaHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaEasternColoradoHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaEasternOklahomaHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaCheyenneHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaShreveportHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaSoutheastLouisianaHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaJacksonHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaHoustonHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaFayettevilleArkansasHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaGulfCoastHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaCentralArkansasHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaWestPalmBeachHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaAlexandriaHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaOrlandoHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaTampaHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaMiamiHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaCaribbeanHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaNorthFloridaHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaBayPinesHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaTuscaloosaHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaDublinHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaColumbiaSouthCarolinaHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaCharlestonHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaCentralAlabamaHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaAugustaHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaBirminghamHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaAtlantaHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaSalisburyHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaSalemHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaRichmondHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaHamptonHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaDurhamHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaFayettevilleCoastalHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaAshevilleHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaWilkesBarreHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaPhiladelphiaHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaWilmingtonHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaCoatesvilleHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaErieHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaLebanonFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaButlerHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__pittsburghHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaAltoonaHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaWesternNewYorkHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaSyracuseHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaNewYorkHarborHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaNorthportHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaNewJerseyHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaHudsonValleyHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaFingerLakesHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaBronxHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaAlbanyHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaWhiteRiverJunctionHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaProvidenceHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaManchesterHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaMaineHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaConnecticutHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaCentralWesternMassachusettsHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaBostonHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetFacilitySidebar__vaBedfordHealthCareFacilitySidebarQuery | 0s | [n/a] pages |
| GetIcsFiles | 0s | [n/a] pages |
| GetSidebars | 0s | [n/a] pages |
| GetCampaignLandingPages | 0s | 0 pages |
| GetNodeBasicLandingPage | 0s | 1 pages |
| GetNodeSupportResourcesDetailPage | 0s | 10 pages |
| GetNodeMediaListVideos | 0s | 0 pages |
| GetNodeChecklist | 0s | 0 pages |
| GetNodeMediaListImages | 0s | 0 pages |
| GetNodeStepByStep | 0s | 6 pages |
| GetNodeMultipleQaPages | 0s | 13 pages |
| GetNodeQa__slice4 | 0s | 1 pages |
| GetNodeQa__slice3 | 0s | 25 pages |
| GetNodeQa__slice2 | 0s | 25 pages |
| GetNodeQa__slice1 | 0s | 25 pages |
| GetNodeHealthCareRegionDetailPage__slice3 | 0s | 13 pages |
| GetNodeHealthCareRegionDetailPage__slice2 | 0s | 50 pages |
| GetNodePublicationListingPages | 0s | 1 pages |
| GetNodeHealthCareRegionDetailPage__slice1 | 0s | 50 pages |
| GetNodeVamcOperatingStatusAndAlerts | 0s | 3 pages |
| GetNodeLocationsListingPages | 0s | 3 pages |
| GetNodeLeadershipListingPages | 0s | 3 pages |
| GetNodeStoryListingPages | 0s | 3 pages |
| GetNodeEvents__slice11 | 0s | 32 pages |
| GetNodeEvents__slice9 | 0s | 50 pages |
| GetNodeEvents__slice10 | 0s | 50 pages |
| GetNodeEvents__slice8 | 0s | 50 pages |
| GetNodeEvents__slice7 | 0s | 50 pages |
| GetNodeEvents__slice6 | 0s | 50 pages |
| GetNodeEvents__slice5 | 0s | 50 pages |
| GetNodeEvents__slice4 | 0s | 50 pages |
| GetNodeEvents__slice3 | 0s | 50 pages |
| GetNodeEvents__slice2 | 0s | 50 pages |
| GetNodeEvents__slice1 | 0s | 50 pages |
| GetNodeEventListingPage | 0s | 4 pages |
| GetNodePressReleaseListingPages | 0s | 3 pages |
| GetNodeOffices | 0s | 1 pages |
| GetNodeNewsStoryPages | 0s | 43 pages |
| GetNodePressReleasePages | 0s | 30 pages |
| GetNodeHealthServicesListingPage__slice1 | 0s | 3 pages |
| GetNodeHealthCareLocalFacilityPages__slice1 | 1s | 21 pages |
| GetNodePersonProfile__slice2 | 1s | 50 pages |
| GetNodePersonProfile__slice3 | 1s | 24 pages |
| GetNodePersonProfile__slice1 | 0s | 50 pages |
| GetNodeVaForm__slice18 | 1s | 13 pages |
| GetNodeVaForm__slice17 | 1s | 25 pages |
| GetNodeVaForm__slice16 | 1s | 25 pages |
| GetNodeHealthCareRegionPage__slice1 | 1s | 3 pages |
| GetNodeVaForm__slice15 | 1s | 25 pages |
| GetNodeVaForm__slice14 | 0s | 25 pages |
| GetNodeVaForm__slice13 | 1s | 25 pages |
| GetNodeVaForm__slice11 | 1s | 25 pages |
| GetNodeVaForm__slice12 | 1s | 25 pages |
| GetNodeVaForm__slice10 | 1s | 25 pages |
| GetNodeVaForm__slice9 | 0s | 25 pages |
| GetNodeVaForm__slice8 | 0s | 25 pages |
| GetNodeVaForm__slice7 | 0s | 25 pages |
| GetNodeVaForm__slice6 | 0s | 25 pages |
| GetNodeVaForm__slice5 | 0s | 25 pages |
| GetNodeVaForm__slice4 | 0s | 25 pages |
| GetNodeVaForm__slice3 | 0s | 25 pages |
| GetNodeVaForm__slice2 | 0s | 25 pages |
| GetNodeVaForm__slice1 | 0s | 25 pages |
| GetNodeLandingPages | 0s | 12 pages |
| GetNodePage__slice11 | 0s | 24 pages |
| GetNodePage__slice10 | 0s | 25 pages |
| GetNodePage__slice9 | 0s | 25 pages |
| GetNodePage__slice8 | 0s | 25 pages |
| GetNodePage__slice6 | 0s | 25 pages |
| GetNodePage__slice7 | 0s | 25 pages |
| GetNodePage__slice5 | 0s | 25 pages |
| GetNodePage__slice4 | 0s | 25 pages |
| GetNodePage__slice3 | 0s | 25 pages |
| GetNodePage__slice2 | 0s | 25 pages |
| GetNodePage__slice1 | 0s | 25 pages |
Finished 168 queries in 11s with 1720 pages
https://prod.cms.va.gov response time: : 12.062s
Total time to load content from GraphQL: 12.452s
Drupal successfully loaded!
```
</details>

## Screenshots
See output

## Acceptance criteria
- [ ] Performance boost

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/site/stages/build/drupal/api.js [hunk 1]
```diff
@@ -248,13 +257,14 @@ function getDrupalClient(buildOptions, clientOptionsArg) {
...
+      const maxParallelRequests = 15;
       const overallStartTime = moment();
       const staggeredRequests = new Array(maxParallelRequests)
         .fill(null)
-        .map(() => {
+        .map((_, index) => {
           return new Promise(resolve => {
-            setTimeout(() => resolve(parallelQuery()), 1000);
+            const delay = index * 250;
+            setTimeout(() => resolve(parallelQuery()), delay);
```
Comment (ncksllvn) [inline src/site/stages/build/drupal/api.js:267]: This adds a "stagger" behavior to initial request execution. They are requested with a 250ms delay between each one, which seems better than just blasting the CMS with 15 all at once.

**Review by ethanteague — APPROVED** [review]
LGTM

File: src/site/stages/build/drupal/graphql/bioPage.graphql.js [hunk 2]
```diff
@@ -57,7 +57,7 @@ function getNodePersonProfilesSlice(operationName, offset, limit) {
       nodeQuery(
         limit: ${limit}
         offset: ${offset}
-        sort: { field: "changed", direction:  ASC }
+        sort: { field: "nid", direction:  ASC }
```
Comment (jhonnyoddball) [inline src/site/stages/build/drupal/graphql/bioPage.graphql.js:60]: Are these changes going to impact the cms export build as far as discrepancies?

**Review by jhonny218 — APPROVED** [review]
LGTM

File: src/site/stages/build/drupal/graphql/healthServicesListingPage.graphql.js [hunk 3]
```diff
@@ -120,7 +120,7 @@ function getNodeHealthServicesListingPages(operationName, offset, limit) {
       nodeQuery(
         limit: ${limit}
         offset: ${offset}
-        sort: { field: "title", direction:  ASC }
+        sort: { field: "nid", direction:  ASC }
```
Comment (swirtSJW) [inline src/site/stages/build/drupal/graphql/healthServicesListingPage.graphql.js:123]: Does this alter the display order, or are these not related?

Comment (ncksllvn) [reply src/site/stages/build/drupal/graphql/healthServicesListingPage.graphql.js:123]: There is logic after the GraphQL response that should do the actual sort. This query didn't have a sort order at all before so I hope we weren't trying to rely on it.

File: src/site/stages/build/drupal/graphql/bioPage.graphql.js (hunk 2 above)
Comment (ncksllvn) [reply src/site/stages/build/drupal/graphql/bioPage.graphql.js:60]: Sorry I missed this. No, unless the Transformers were relying on an exact array order from the raw, unsorted GraphQL response and I can't imagine that was the case?
